superset/contrib/docker/superset_config.py

```diff
@@ -122,7 +122,7 @@ class CeleryConfig(object):
 
 # Defaults configuration
 
-DEFAULT_DASHBOARD_ID = 2 if SYNC_MODE else 1
+DEFAULT_DASHBOARD_ID = 2 if SYNC_MODE else 2
```

Comment:
`DEFAULT_DASHBOARD_ID = 2`

Reply:
true, but this way you can see where it need to be set, if it's not the same ID in the future

Reply:
um. I kind of disagree. We just need to make sure that default is 2 always. But I don't think I'm going to block the PR because of it now.